scripts/mainmenu_mapselector.js

```diff
@@ -213,25 +215,27 @@ class MapSelection {
 
 		$.persistentStorage.setItem('mapSelector.filtersToggled', this.filtersToggled);
 
-		this.filtersPanel.SetHasClass('mapselector-filters--filters-extended', this.filtersToggled);
+		this.panels.filtersPanel.SetHasClass('mapselector-filters--filters-extended', this.filtersToggled);
 	}
 
 	/**
 	 * Clear all the filters, resetting to the default state
 	 */
 	static clearFilters() {
 		// Uncheck every checked gamemode button
-		Object.keys(MapSelection.gameModeData).forEach((mode) => {
-			let button = MapSelection.gameModeData[mode].filterButton;
+		Object.keys(this.gameModeData).forEach((mode) => {
+			let button = this.gameModeData[mode].filterButton;
 			if (button && !button.checked) {
 				button.SetSelected(true);
 			}
 		});
 
 		// Reset every NState button
-		[this.completedFilterButton, this.favoritesFilterButton, this.downloadedFilterButton].forEach((button) => {
-			button.currentstate = 0;
-		});
+		[
+			this.panels.completedFilterButton,
+			this.panels.favoritesFilterButton,
+			this.panels.downloadedFilterButton
+		].forEach((button) => (button.currentstate = 0));
 
 		// Reset tier slider
 		this.tierSlider.SetValues(TIER_MIN, TIER_MAX);
```

Comment:
```suggestion
		this.panels.tierSlider.SetValues(TIER_MIN, TIER_MAX);
```

Reply:
Honestly go through this file and triple check and make sure all the panels are properly referenced by the `this.panels.` notation now.

Reply:
Yikes, sorry I did such a crap job on this. Been through everything thoroughly this time.